pom.xml

```diff
@@ -10,6 +10,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <dw.version>0.7.1</dw.version>
+    <advanced-dropwizard-healthcheck.version>4.14.1</advanced-dropwizard-healthcheck.version>
```

Comment:
Needs to be updated to latest point release.

Reply:
Right, thx.